tests/testthat/test-HiSCR.R

```diff
@@ -0,0 +1,61 @@
+test_that("check_abscess_count_increase", {
+  expect_false(all(calculate_hiscr(c(3, 2, 4), c(5, 4, 6), c(2, 1, 3),
```

Comment:
Since this is supposed to calculate three values, it would be preferable to use:

```
expect_equal(
  calculate_hiscr(...),
  c(FALSE, FALSE, FALSE)
)
```

The test as written would succeed if one of the values is false and the other two are TRUE (which I don't think is the intent).

Please apply this comment throughout.

Reply:
Replaced with #11